libs/web/include/mrpt/web/CWebSocket.h

```diff
@@ -1,17 +1,94 @@
-#include <iostream>
+#include <boost/beast/core.hpp>
```

Comment:
Perhaps there is a way to remove boost as an external dependency and hide it behind an interface. Internal dependencies are fine.

For example,
https://github.com/MRPT/mrpt/blob/0058c3cc7ad7212b5bee5600789190b5c9bb80eb/libs/hwdrivers/include/mrpt/hwdrivers/CFFMPEG_InputStream.h
https://github.com/MRPT/mrpt/blob/0058c3cc7ad7212b5bee5600789190b5c9bb80eb/libs/hwdrivers/src/CFFMPEG_InputStream.cpp
mrpt::pimpl prevents ffmpeg from being an external dependency, but it's fine to be an internal dependency.